schematic/models/metadata.py

```diff
@@ -178,34 +181,13 @@ def validateModelManifest(self, manifestPath: str, rootNode: str, jsonSchema: st
         # get annotations from manifest (array of json annotations corresponding to manifest rows)
         manifest = pd.read_csv(manifestPath).fillna("")
 
-
```

Comment:
Note: this flogic needs to stay in develop. 

@sujaypatil96 could you apply your changes on the latest code in develop?